src/test/java/uk/gov/register/functional/EntryResourceFunctionalTest.java

```diff
@@ -28,21 +29,23 @@
 
 public class EntryResourceFunctionalTest {
     @ClassRule
-    public static RegisterRule register = new RegisterRule("address");
+    public static RegisterRule register = new RegisterRule();
     private final String item1 = "{\"address\":\"6789\",\"street\":\"elvis\"}";
     private final String item2 = "{\"address\":\"6790\",\"street\":\"presley\"}";
     private final String item1Hash = "sha-256:" + DigestUtils.sha256Hex(item1);
     private final String item2Hash = "sha-256:" + DigestUtils.sha256Hex(item2);
-    
+    private WebTarget addressTarget;
+
     @Before
     public void publishTestMessages() throws Throwable {
         register.wipe();
-        register.mintLines(item1, item2);
+        register.mintLines("address", item1, item2);
+        addressTarget = register.targetRegister("address");
```

Comment:
If the previous comment in `EntriesResourceFunctionalTest.java` didn't infuriate you maybe we can do the same for this class?